static/sass/styles.scss

```diff
@@ -108,7 +108,7 @@
   }
 }
 
-.p-stepped-list__item:first-child {
+.p-list__item:first-child {
```

Comment:
Why was this changed? Doesn't it break something that this style was doing? I think it was supposed to hide the line before first element in stepped list? Is this needed on `p-list__item` but not `p-stepped-list__item`?